WordPressKit.podspec

```diff
@@ -21,8 +21,11 @@ Pod::Spec.new do |s|
 
   s.dependency 'Alamofire', '~> 4.8.0'
   s.dependency 'CocoaLumberjack', '~> 3.4'
-  s.dependency 'WordPressShared', '~> 1.15.0'
   s.dependency 'NSObject-SafeExpectations', '0.0.4'
   s.dependency 'wpxmlrpc', '~> 0.9'
   s.dependency 'UIDeviceIdentifier', '~> 1.4'
+
+    # Use a loose restriction that allows both production and beta versions, up to the next major version.
+  # If you want to update which of these is used, specify it in the host app.
+  s.dependency 'WordPressShared', '~> 1.15-beta' # Don't change this until we hit 2.0 in WPShared
```

Comment:
We can definitely try this, but I foresee people just updating the shared version in WPKit when this doesn't work (instead of asking questions) 🤷‍♀️